### PR TITLE
refactor: move helm README to docs/helm.md

### LINKED ISSUE #3877: Move helm docs to /docs
Move helm docs to /docs so that we can display on the `coder.com/docs/code-server` website.

> Yes, if we want to add it to the docs page, we have to move it to live inside of the docs folder.
https://github.com/cdr/code-server/pull/3869#discussion_r679332665

See prior discussion: https://github.com/cdr/code-server/pull/3869#discussion_r678810300

### PR DESCRIPTION
**Pull request by jsjoeio**
Moves `README` with helm information into `docs/helm.md` so that it can be rendered and read on the docs website.

Fixes #3877

### CODE REVIEW

File: docs/helm.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# code-server
+# code-server Helm
```
Comment (jawnsy) [inline docs/helm.md:1]:
```suggestion
# code-server Helm Chart
```

File: docs/install.md [hunk 2]
```diff
@@ -224,7 +224,7 @@ alternative](https://hub.docker.com/r/linuxserver/code-server).
 
 ## Helm
 
-You can install code-server via [Helm](https://github.com/cdr/code-server/blob/main/ci/helm-chart/README.md).
+You can install code-server via [Helm](https://coder.com/docs/code-server/latest/helm).
```
Comment (jawnsy) [inline docs/install.md:227]:
```suggestion
You can install code-server using the [Helm package manager](https://coder.com/docs/code-server/latest/helm).
```

File: docs/manifest.json [hunk 3]
```diff
@@ -37,6 +37,11 @@
           "description": "How to run code-server --link",
           "path": "./link.md"
         },
+        {
+          "title": "Helm",
+          "description": "How to install code-server via Helm",
```
Comment (jawnsy) [inline docs/manifest.json:42]:
```suggestion
          "description": "How to install code-server using the Helm package manager",
```

**Review by jawnsy — APPROVED** [review]

**Comment by jsjoeio** [comment]
lol @jawnsy can you tell I've never used Helm? 🤣

**Comment by jawnsy** [comment]
@jsjoeio We all have to start somewhere :)

**Review by TeffenEllis — APPROVED** [review]

**Pull request merged**